app/src/main/java/com/github/sdp_begreen/begreen/firebase/models/FirebaseUser.kt

```diff
@@ -1,15 +1,16 @@
 package com.github.sdp_begreen.begreen.firebase.models
 
 import com.github.sdp_begreen.begreen.models.PhotoMetadata
+import com.github.sdp_begreen.begreen.models.ProfilePhotoMetadata
 import com.github.sdp_begreen.begreen.models.User
 
 /**
  * This class adapts the class User such that it can be effectively stored to Firebase
  */
 data class FirebaseUser (val id: String, var score: Int, val displayName: String? = null, var rating: Int = 0,
-                         var img: PhotoMetadata? = null, var description: String? = null, var phone: String? = null,
+                         var img: ProfilePhotoMetadata? = null, var description: String? = null, var phone: String? = null,

```

Comment:
I noticed and I I left a question about it but it was probably lost. I didn't want to remove it since it could have an implication in the future. Glad you noticed it too, I've removed it !